插件依赖 "@tarojs/plugin-platform-h5" 加载失败，请检查插件配置

<!-- 请不要删除自动生成的 Issue 标签 -->
<!-- 请不要删除自动生成的 Issue 标签 -->
### 相关平台
微信小程序


**小程序基础库: 无**
**使用框架: Vue 3**

### 复现步骤
初始化一个新的taro3.6.2版本的项目yarn dev:h5 命令报错
Error: Cannot find module 'tslib'
Require stack:
- E:\taro\WechatAppDemo\node_modules\@tarojs\plugin-platform-h5\dist\index.js
- E:\taro\WechatAppDemo\node_modules\@tarojs\plugin-platform-h5\index.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\utils\index.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\Kernel.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\index.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\index.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\dist\cli.js
- C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\bin\taro
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Function.Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at Object.<anonymous> (E:\taro\WechatAppDemo\node_modules\@tarojs\plugin-platform-h5\dist\index.js:5:13)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1308:10)
    at Object.newLoader [as .js] (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Function.Module._load (node:internal/modules/cjs/loader:958:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'E:\\taro\\WechatAppDemo\\node_modules\\@tarojs\\plugin-platform-h5\\dist\\index.js',
    'E:\\taro\\WechatAppDemo\\node_modules\\@tarojs\\plugin-platform-h5\\index.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\node_modules\\@tarojs\\service\\dist\\utils\\index.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\node_modules\\@tarojs\\service\\dist\\Kernel.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\node_modules\\@tarojs\\service\\dist\\index.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\node_modules\\@tarojs\\service\\index.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\dist\\cli.js',
    'C:\\Users\\ruler2358\\AppData\\Roaming\\npm\\node_modules\\@tarojs\\cli\\bin\\taro'
  ]
}
C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\utils\index.js:75
                    throw new Error(`插件依赖 "${item}" 加载失败，请检查插件配置`);
                          ^

Error: 插件依赖 "@tarojs/plugin-platform-h5" 加载失败，请检查插件配置
    at apply (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\src\utils\index.ts:72:17)
    at Kernel.initPlugin (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\src\Kernel.ts:157:5)
    at Kernel.resolvePlugins (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\src\Kernel.ts:130:12)
    at Kernel.initPresetsAndPlugins (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\src\Kernel.ts:115:10)
    at Kernel.<anonymous> (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\src\Kernel.ts:312:10)
    at Generator.next (<anonymous>)
    at C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\Kernel.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\Kernel.js:4:12)
    at Kernel.run (C:\Users\ruler2358\AppData\Roaming\npm\node_modules\@tarojs\cli\node_modules\@tarojs\service\dist\Kernel.js:247:16)

Node.js v18.15.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

### 期望结果
编译成功

### 实际结果
初始化项目编译失败

### 环境信息
```
👽 Taro v3.6.2


  Taro CLI 3.6.2 environment info:
    System:
      OS: Windows 10
    Binaries:
      Node: 18.15.0 - E:\node\node.EXE
      Yarn: 1.22.19 - C:\Users\ruler2358\AppData\Roaming\npm\yarn.CMD
      npm: 9.5.1 - E:\node\npm.CMD

```

<!-- generated by taro-issues. 请勿修改或删除此行注释 --><!--labels=T-weapp,V-3,F-vue3-->

--- a/packages/taro-components-advanced/rollup.config.js
+++ b/packages/taro-components-advanced/rollup.config.js
@@ -22,7 +22,7 @@ export default {
   treeshake: false,
   plugins: [
     externals({
-      devDeps: true
+      devDeps: false
     }),
     resolve({
       preferBuiltins: false,

--- a/packages/taro-components-library-react/rollup.config.js
+++ b/packages/taro-components-library-react/rollup.config.js
@@ -16,7 +16,7 @@ const config = {
   treeshake: false,
   plugins: [
     externals({
-      devDeps: true
+      devDeps: false
     }),
     resolve({
       preferBuiltins: false,

--- a/packages/taro-components-library-vue2/rollup.config.js
+++ b/packages/taro-components-library-vue2/rollup.config.js
@@ -16,7 +16,7 @@ const config = {
   treeshake: false,
   plugins: [
     externals({
-      devDeps: true,
+      devDeps: false,
       exclude: ['vue-fragment']
     }),
     resolve({

--- a/packages/taro-components-library-vue3/rollup.config.js
+++ b/packages/taro-components-library-vue3/rollup.config.js
@@ -16,7 +16,7 @@ const config = {
   treeshake: false,
   plugins: [
     externals({
-      devDeps: true
+      devDeps: false
     }),
     resolve({
       preferBuiltins: false,

--- a/packages/taro-components-react/rollup.config.js
+++ b/packages/taro-components-react/rollup.config.js
@@ -10,7 +10,7 @@ export default {
   input: 'src/index.ts',
   plugins: [
     externals({
-      devDeps: true,
+      devDeps: false,
       include: [/^react$/]
     }),
     resolve({

--- a/packages/taro-platform-h5/rollup.config.js
+++ b/packages/taro-platform-h5/rollup.config.js
@@ -40,7 +40,7 @@ const variesConfig = [{
     format: 'cjs'
   },
   plugins: getPlugins([externals({
-    devDeps: true
+    devDeps: false
   })])
 }, {
   input: [
@@ -54,7 +54,7 @@ const variesConfig = [{
     preserveModulesRoot: 'src'
   },
   plugins: getPlugins([externals({
-    devDeps: true
+    devDeps: false
   })])
 }, {
   input: path.join(cwd, 'src/runtime/apis/index.ts'), // 供 babel-plugin-transform-taroapi 使用，为了能 tree-shaking
@@ -65,7 +65,6 @@ const variesConfig = [{
   },
   plugins: getPlugins([
     externals({
-      devDeps: true,
       exclude: ['@tarojs/components', '@tarojs/taro-h5']
     })
   ], [exportNameOnly()])

--- a/packages/taro-plugin-html/rollup.config.js
+++ b/packages/taro-plugin-html/rollup.config.js
@@ -6,7 +6,7 @@ const cwd = __dirname
 
 const base = {
   plugins: [
-    externals({ devDeps: true }),
+    externals({ devDeps: false }),
     ts(),
   ]
 }

--- a/packages/taro-plugin-http/package.json
+++ b/packages/taro-plugin-http/package.json
@@ -40,10 +40,10 @@
     "jest-cli": "^29.3.1",
     "jest-environment-node": "^29.5.0",
     "rollup": "^3.8.1",
-    "ts-jest": "^29.0.5",
-    "tslib": "^2.4.0",
     "rollup-plugin-node-externals": "^5.0.0",
     "rollup-plugin-ts": "^3.0.2",
+    "ts-jest": "^29.0.5",
+    "tslib": "^2.4.0",
     "typescript": "^4.7.4"
   }
 }

